src/main/java/br/edu/ufape/agiota/negocio/services/ClienteService.java

```diff
@@ -0,0 +1,79 @@
+package br.edu.ufape.agiota.negocio.services;
+
+import br.edu.ufape.agiota.negocio.basica.Cliente;
+import br.edu.ufape.agiota.negocio.repositorios.ClienteRepository;
+import br.edu.ufape.agiota.negocio.services.interfaces.UsuarioServiceInterface;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class ClienteService implements UsuarioServiceInterface {
+
+    @Autowired
+    private ClienteRepository clienteRepository;
+
+    public List<Cliente> usuarios() {
+        return clienteRepository.findAll();
+    }
+
+    public Cliente create(Cliente cliente) throws Exception {
+        if(findByEmail(cliente.getEmail()) == null)
```

Comment:
```suggestion
        if(findByEmail(isNull(cliente.getEmail()))
```
Substituir pela verificação do java.util.Objects